extensions/interactions/base.py

```diff
@@ -91,6 +91,11 @@ class BaseInteraction(object):
     # Customization arg specifications for the component, including their
     # descriptions, schemas and default values. Overridden in subclasses.
     _customization_arg_specs = []
+    # Instructions for using this interaction, to be shown to the learner. Only
+    # relevant for supplemental interactions.
+    instructions = None
+    # Whether the answer is long, and would benefit from being summarized.
+    needs_summary = False
```

Comment:
(For now, I plan to leave this as-is. It doesn't seem to me obvious a priori that these two things are necessarily coupled, and it shouldn't be hard to change later if it turns out that they are.)